src/devices/Common/Iot/Device/Common/WeatherHelper.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Units;
+using System;
+
+namespace Iot.Device.Common
+{
+    /// <summary>
+    /// Helpers for weather. Formulas taken from http://www.reahvac.com/tools/humidity-formulas/.
```

Comment:
Link should not be in the xml docs since this will be eventually displayed on the docs page and it's not exactly interesting for users and rather for maintainers. Move the link anywhere but xml docs :smile:

Reply:
Done.